lib/WUI/netdev.h

```diff
@@ -126,8 +126,12 @@ netdev_status_t netdev_get_status(uint32_t);
 ////////////////////////////////////////////////////////////////////////////
 /// @brief Retrive method used to obtain ip address on active network device
 ///
+/// @param[in] dev_id device ID. One of
+///             - #NETDEV_ETH_ID
+///             - #NETDEV_ESP_ID
+///             - #NETDEV_NODEV_ID
```

Comment:
:bomb: I think passing the `NETDEV_NODEV_ID` will result in out of bounds access on the array which has only 2 items.